src/view/ui.py

```diff
@@ -146,7 +146,7 @@ def setWidget(self, widget):
         ----------
         widget: QWidget
         """
-        for k, v in widget.__dict__.items():
+        for k in widget.__dict__:
             if k in self.__dict__:
```

Comment:
In case of comparing only dict keys, you can also make the difference of sets of keys, to obtain all the keys that differ between both dict, and maybe not only print the first one that is in the two objects?

Reply:
good catch!

Reply:
Nice 👍